content/ri/installing/install-k8s.md

```diff
@@ -9,210 +9,203 @@ nextStep:
-          mountPath: /db
-        ports:
-        - containerPort: 8001 #exposed container port and protocol
-          protocol: TCP
-      volumes:
-      - name: db
-        emptyDir: {} # node-ephemeral volume https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
-```
-
-2. Create the RedisInsight deployment and service
-
-```sh
-kubectl apply -f redisinsight.yaml
-```
-
-3. Once the deployment and service are successfully applied and complete, access RedisInsight. This can be accomplished by listing the using the `<external-ip>` of the service we created to reach redisinsight.
-
-```sh
-$ kubectl get svc redisinsight-service
-NAME                   CLUSTER-IP       EXTERNAL-IP      PORT(S)         AGE
-redisinsight-service   <cluster-ip>     <external-ip>    80:32143/TCP    1m
-```
-
-4. If you are using minikube, run `minikube list` to list the service and access RedisInsight at `http://<minikube-ip>:<minikube-service-port>`.
-```
-$ minikube list
-|-------------|----------------------|--------------|---------------------------------------------|
-|  NAMESPACE  |         NAME         | TARGET PORT  |           URL                               |
-|-------------|----------------------|--------------|---------------------------------------------|
-| default     | kubernetes           | No node port |                                             |
-| default     | redisinsight-service |           80 | http://<minikube-ip>:<minikubeservice-port> |
-| kube-system | kube-dns             | No node port |                                             |
-|-------------|----------------------|--------------|---------------------------------------------|
-```
+The example below is a RedisInsight deployment file with one [replica](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment) and [ephemeral storage](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). The following sections will show you how to [add a load balancer service]({{<relref "/ri/installing/install-k8s.md#create-a-redisinsight-service">}}) or [persistent storage claim]({{<relref "/ri/installing/install-k8s.md#create-a-persistent-volume-claim">}}).
 
-## Create the RedisInsight deployment without a service.
+Make sure to substitute your own values into your deployment file. Go to [kubernetes.io](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) for more information on deployments.
 
-Below is an annotated YAML file that will create a RedisInsight
-deployment in a K8s cluster.
-
-1. Create a new file redisinsight.yaml with the content below
+{{<note>}}If you have a service named `redisinsight` that exposes the deployment, it may cause environment variable conflicts with the RedisInsight software. You can manually override the service environnement variables (as seen in this example) or change the service name (as seen in the service example below).{{</note>}}
```

Comment:
```suggestion
{{<note>}}If you have a service named `redisinsight` that exposes the deployment, it may cause environment variable conflicts with the RedisInsight software. You can manually override the service environment variables (as seen in this example) or change the service name (as seen in the service example below).{{</note>}}
```